src/jvmMain/kotlin/io/beatmaps/api/review.kt

```diff
@@ -491,6 +502,7 @@ fun Route.reviewRoute() {
             val update = call.receive<ReplyRequest>()
 
             captchaIfPresent(update.captcha) {
+                var updated = false
```

Comment:
Seems unnecessary when this is just `response.success`

Reply:
Definitely! This is so much smarter.